### ISSUE TITLE
Updated output documentation for yak triobin

### ISSUE THREAD

**Issue by williamrowell** [lh3/yak#1]
https://github.com/lh3/yak/blob/6de3affe265bb0508ed6d78b36f121bdaf796f71/triobin.c#L176

Do you have any updated documentation for the output of `yak triobin`?  I'm looking at the output of verison r43, which has 13 columns as opposed to the 10 columns documented in the help text.  I'm especially trying to understand column 2, which has values `m`, `p`, `a`, and `0`.

**Comment by lh3** [comment]
* m=mother
* p=father
* a=ambiguous, 0=noCount/ambiguous

**Comment by williamrowell** [comment]
Thanks for the quick answer!  That's what I guessed, but wanted to make sure before proceeding.  Thanks for the tool!

**Comment by lh3** [comment]
Forgot to say that you can ignore most of other columns. Those are mostly for debugging purpose.

**Comment by zeeev** [comment]
Dear @lh3,

We are testing out trio binning and it looks like our binned assemblies are more fragmented than the non-binned assemblies. Both haplotypes have good coverage. Is there a way to adjust the triobinning step to be more specific? I.E. require more p/m markers? 

What is the meaning of these options?:
```
  -c INT     min occurrence [2]
  -d INT     mid occurrence [5]
```

**Comment by lh3** [comment]
By default, if a k-mer occurs 5 times or more in mother but occurs twice or less in father, the k-mer is considered to be a mother-specific k-mer. The label on the 2nd column is determined by the rest of columns under complex rules coded in function `tb_classify()`. You can't tune these rules on the command line.

It is hard to get perfect trio binning. Hifiasm effectively uses the HiFi assembly graph to fix binning errors. Without doing that, hifiasm would only get ~10Mb N50, comparable to trio HiCanu.

**Comment by lh3** [comment]
For a simple way to increase specificity:
```
awk '$3>=21&&$4<=2&&$2=="p"' triobin.txt > paternal.txt
awk '$4>=21&&$3<=2&&$2=="m"' triobin.txt > maternal.txt
# the rest are ambiguous
```

**Comment by zeeev** [comment]
Hi @lh3,

Thank you for sharing these ideas. Just confirming, you think triobinning isn't as effective as just assembling and phasing in a single genome? That has been my experience, at least using yak HifiASM and-or IPA.

**Comment by lh3** [comment]
Yes, when HiFi phasing and trio phasing are inconsistent, HiFi phasing is often the correct one.

**Comment by lh3** [comment]
In early days, we tried hicanu trio binning. I manually inspected many differences between hicanu and yak binning. I think yak is generally more accurate. Nonetheless, the assembly with hicanu binning is similar to the assembly with yak binning.

**Comment by lh3** [comment]
Also, hifiasm applies trio binning to error corrected reads. This noticeably improves the binning accuracy: there are much fewer inconsistencies between trio phasing and hifi read phasing.
